pages/linux/sbatch.md

```diff
@@ -0,0 +1,19 @@
+# sbatch
+
+> Submit a batch job to the SLURM scheduler.
+
+- Submit a batch job by file:
```

Comment:
I don't think "by file" is necessary here. Since I don't see any other ways of submitting jobs, and the file is pretty apparent here.